website/docs/FAQ.md

```diff
@@ -132,6 +132,12 @@ Workaround:
 
 Explanation: Per [this gist](https://gist.github.com/defulmere/8b9695e415a44271061cc8e272f3c300?permalink_comment_id=4711478#gistcomment-4711478), linked from the official [chromadb docs](https://docs.trychroma.com/troubleshooting#sqlite), adding this folder triggers chromadb to use pysqlite3 instead of the default.
 
+## How to register a reply function
+
+(from [issue #478](https://github.com/microsoft/autogen/issues/478))
```

Comment:
What about copying the example from #478 to here?
@victordibia @gagb what do you think?

Reply:
I think directly show the example would be good for readers

Reply:
Example added